wandb/sdk/artifacts/storage_handlers/s3_handler.py

```diff
@@ -266,12 +266,13 @@ def _etag_from_obj(obj: "boto3.s3.Object") -> ETag:
         etag = obj.e_tag[1:-1]  # escape leading and trailing quote
         return etag
 
-    @staticmethod
-    def _extra_from_obj(obj: "boto3.s3.Object") -> Dict[str, str]:
+    def _extra_from_obj(self, obj: "boto3.s3.Object") -> Dict[str, str]:
```

Comment:
So the type annotation is incorrect here, right?

Reply:
Oh, it is, thanks for pointing out.